src/frontends/onnx/tests/__init__.py

```diff
@@ -168,3 +168,7 @@ def xfail_test(reason="Mark the test as expected to fail", strict=True):
                                        "Not equal to tolerance")
 xfail_issue_122775 = xfail_test(reason="test_resize_downsample_scales_linear_cpu - "
                                        "Not equal to tolerance")
+
+skip_issue_127649 = pytest.mark.skip(reason="Not equal to tolerance rtol=0.001, atol=1e-07 - "
```

Comment:
need to skip it instead of xfailing it because it works on Azure and does not work on GHA. As both workflows are working, the coverage would not drop.